modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -218,6 +216,8 @@ def prepare_appointment(appointment)
 
         appointment[:station], appointment[:ien] = extract_station_and_ien(appointment)
 
+        appointment[:minutes_duration] ||= 60 if appointment[:appointment_type] == 'COMMUNITY_CARE'
```

Comment:
should we carry over the check to see if minutes duration is nil before overwriting it?

Reply:
This only overwrites it if it is nil

Reply:
oh right. I missed that.